core/src/main/java/org/elasticsearch/search/builder/SearchSourceBuilder.java

```diff
@@ -91,7 +91,7 @@
     public static final ParseField SORT_FIELD = new ParseField("sort");
     public static final ParseField TRACK_SCORES_FIELD = new ParseField("track_scores");
     public static final ParseField INDICES_BOOST_FIELD = new ParseField("indices_boost");
-    public static final ParseField AGGREGATIONS_FIELD = new ParseField("aggregations", "aggs");
+    public static final ParseField AGGREGATIONS_FIELD = new ParseField("aggregations", new String[] { "aggs" }, Strings.EMPTY_ARRAY);
```

Comment:
I need it to force it to select the three arg constructor rather than the two arg one which would cause `"aggs"` to be deprecated instead of set as an alternative. This is what I meant in the commit message about it being intentionally slightly awkward